setup.py

```diff
@@ -69,6 +69,7 @@
         "websockets": ["websockets>=12.0,<13"],
         "jupyter-executor": jupyter_executor,
         "types": ["mypy==1.9.0", "pytest>=6.1.1,<8"] + jupyter_executor,
+        "long-context": ["llmlingua<0.3"],
```

Comment:
How about documenting this option in https://microsoft.github.io/autogen/docs/installation/Optional-Dependencies ?